### PR TITLE
Move config api to stable

### PR DESCRIPTION
**Pull request by shivamkm07**
# Description

This PR updates config API to stable. It updates the protos to include the stable apis. There has been some workflow changes causing the build to fail with latest dapr protos. It also fixes it

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .github/workflows/itests.yml [hunk 1]
```diff
@@ -41,21 +41,21 @@ jobs:
...
       DAPR_CLI_REF: ''
-      DAPR_REF: '82e097134cdc7494c5c3be71bf80f7904d6db9ea'
+      DAPR_REF: 'dbeea96710d38d08dcc5aaf6807914a7d697bed4'
     steps:
       - name: Set up Dapr CLI
         run: wget -q ${{ env.DAPR_INSTALL_URL }} -O - | /bin/bash -s ${{ env.DAPR_CLI_VER }}
       - name: Set up Go ${{ env.GOVER }}
         if: env.DAPR_REF != '' || env.DAPR_CLI_REF != ''
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
```
Comment (halspang) [inline .github/workflows/itests.yml:58]: Can we do it like this instead? Then we don't need to keep updating the go version. You'll have to move this step to in between pulling and building dapr.

https://github.com/dapr/test-infra/blob/master/.github/workflows/dapr-longhaul-nightly.yml#L58

File: src/Dapr.Client/DaprClientGrpc.cs [hunk 2]
```diff
@@ -1526,7 +1526,7 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
             {
                 InstanceId = instanceId,
                 WorkflowComponent = workflowComponent,
-                WorkflowType = workflowName //TODO: Change 'WorkflowType' to 'WorkflowName' once changes go through dapr/dapr
+                WorkflowName = workflowName
```
Comment (halspang) [inline src/Dapr.Client/DaprClientGrpc.cs:1529]: This change and the one below it just come from updating the protos?

**Review by halspang — COMMENTED** [review]
Two small comments.

File: src/Dapr.Client/DaprClientGrpc.cs (hunk 2 above)
Comment (shivamkm07) [reply src/Dapr.Client/DaprClientGrpc.cs:1529]: Yes, updating the protos was failing the tests initially. On debugging I found i is due to these renaming in protos, so had to do here as well.

**Pull request merged**